feature-libs/storefinder/styles/layout/_index.scss

```diff
@@ -1,16 +1,16 @@
 @import './store-finder-page';
 
-// The blacklist can be used to prevent template CSS
+// The blocklist can be used to prevent template CSS
 // being generated.
-$page-template-blacklist-store-finder: () !default;
+$page-template-blocklist-store-finder: () !default;
 
-// whitelisted page template selectors are processed
-// unless (some) they are blacklisted
-$page-template-whitelist-store-finder: StoreFinderPageTemplate !default;
+// allowlisted page template selectors are processed
+// unless (some) they are blocklisted
```

Comment:
[nitpick] could you change this to `// unless (some) that are blocklisted using `$page-template-blocklist``